source/deployment/cluster.rst

```diff
@@ -1,116 +1,116 @@
-High Availability (E20) - Beta
+High Availability Cluster (E20)
 ===============================
 
-*Available in Enterprise Edition E20.*
+The high availability cluster feature enables a Mattermost system to maintain service during outages and hardware failures through the use of redundant infrastructure. It is available only in Enterprise Edition E20. 
 
-High Availability support enables a Mattermost server to scale and maintain service during outages and hardware failures through the use of redundant infrastructure. 
+High availability in Mattermost consists of running redundant Mattermost application servers, redundant database servers, and redundant load balancers. The failure of any one of these components does not interrupt operation of the system.
 
-- Support for high availability database configurations is available in Mattermost Enterprise Edition E20 version 3.1 and higher. 
-- Support for high availability configurations of the Mattermost application server is available in Mattermost Enterprise Edition E20 version 3.3 and higher. 
+.. contents::
+  :backlinks: top
+  :local:
+  
+Continuous Operation During Server Updates
+------------------------------------------
 
-This documentation provides a deployment guide, configuration and compatibility details, as well as troubleshooting advice.
+To ensure uninterrupted service during updates, you must make sure that the components are properly sized and that you follow the correct sequence for updating the system components.
```

Comment:
"ensure" is incorrect, should be something like "enable"--the step here is a necessary, but not sufficient condition.